addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6178,6 +6174,17 @@ registry.ImageTools = ImageHandlerOption.extend({
      */
     async _writeShape(svgText) {
         const img = this._getImg();
+        const dataURL = await this.computeShape(svgText, img);
+        return loadImage(dataURL, img);
+    },
+    /**
+     * Sets the image in the supplied SVG and replace the src with a dataURL
+     *
+     * @param {string} svgText svg file as text
+     * @returns {Promise} resolved once the svg is properly loaded
+     * in the document
+     */
+    async computeShape(svgText, img) {
```

Comment:
Since we're introducing both, why not stick to the same name for the parameters? ^^